org.eclipse.scanning.event/src/org/eclipse/scanning/event/ConsumerImpl.java

```diff
@@ -59,8 +61,8 @@
 
 final class ConsumerImpl<U extends StatusBean> extends AbstractQueueConnection<U> implements IConsumer<U> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Override the "equals" method in this class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2160)